### ISSUE TITLE
No Request-Context for cors fetch

### ISSUE THREAD

**Issue by StefanSchoof** [Microsoft/ApplicationInsights-JS#744]
Using beta10 and a config with:
```
           disableFetchTracking: false,
           enableCorsCorrelation: true,
```
reading the [documentation](https://www.npmjs.com/package/@microsoft/applicationinsights-web) for `enableCorsCorrelation` I expect a `Request-Context` Header in my request but I found only a `Request-Id`.

**Comment by markwolff** [comment]
This was intentionally disabled. I think all of the work pending for it is done, so I'll reenable it for next version

**Comment by markwolff** [comment]
fixed in #746
